code/clue.js

```diff
@@ -1,59 +1,359 @@
+  occupation: "Chef",
+  favouriteWeapon: "rope",
+};
+
+const yvette = {
+  firstName: "Miss",
+  lastName: "Yvette",
+  color: "brown",
+  description: "Is probably the most evil villain in the house",
+  age: 36,
+  image: "image/killer/yvette.jpg",
+  occupation: "Maid",
+  favouriteWeapon: "poison",
+};
+
+// STEP 2 - WEAPON
+
+const axe = {
+  name: "Axe",
+  weight: "8 kg",
+  image: "image/weapon/axe.png",
+};
+
+const bat = {
+  name: "Baseball bat",
+  weight: "2 kg",
+  image: "image/weapon/bat.png",
+};
+
+const candlestick = {
+  name: "Candlestick",
+  weight: "1 kg",
+  image: "image/weapon/candlestick.png",
+};
+
+const dumbell = {
+  name: "Dumbell",
+  weight: "10 kg",
+  image: "image/weapon/dumbell.png",
+};
+
+const gun = {
+  name: "Gun",
+  weight: "0,5 kg",
+  image: "image/weapon/gun.png",
+};
+
+const knife = {
+  name: "Knife",
+  weight: "300 g",
+  image: "image/weapon/knife.png",
+};
+
+const posion = {
+  name: "Poison",
+  weight: "100 g kg",
+  image: "image/weapon/poison.png",
+};
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: "Rope",
+  weight: "1 kg",
+  image: "image/weapon/rope.png",
+};
+
+const trophy = {
+  name: "Trophy",
+  weight: "4 kg",
+  image: "image/weapon/trophy.png",
+};
+
+// STEP 3 - ROOM
+
+const diningRoom = {
+  name: "Dining Room",
+  image: "image/room/dining.jpg",
+};
+
+const conservatory = {
+  name: "The Conservatory",
+  image: "image/room/conservatory.png",
+};
+const kitchen = {
+  name: "Kitchen",
+  image: "image/room/kitchen.jpg",
+};
+
+const study = {
+  name: "Study Room",
+  image: "image/room/study.png",
+};
+
+const library = {
+  name: "The Library",
+  image: "image/room/library.jpg",
+};
+
+const billiardRoom = {
+  name: "Billiard Room",
+  image: "image/room/biliard.png",
+};
+
+const lounge = {
+  name: "Lounge",
+  image: "image/room/lounge.jpg",
+};
+
+const ballroom = {
+  name: "Ballroom",
+  image: "image/room/ballroom.jpg",
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const hall = {
+  name: "Hall",
+  image: "image/room/hall.png",
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const spa = {
+  name: "The Spa",
+  image: "image/room/spa.jpg",
+};
+
+const livingRoom = {
+  name: "Living Room",
+  image: "image/room/livingroom.png",
+};
+
+const observatory = {
+  name: "Observatory",
+  image: "image/room/observatory.jpg",
+};
+
+const theatre = {
+  name: "Theatre",
+  image: "image/room/theatre.png",
+};
+
+const guestHouse = {
+  name: "Guest House",
+  image: "image/room/guesthouse.png",
+};
+
+const patio = {
+  name: "Patio",
+  image: "image/room/patio.jpg",
+};
+
+// ARRAY GROUPS
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  mrsWhite,
+  plum,
+  mrsPeacock,
+  wadsworth,
+  colonelMustard,
+  scarlet,
+  yvette,
+];
 
-const weapons = []
+const weaponry = [
+  axe,
+  bat,
+  candlestick,
+  dumbell,
+  gun,
+  knife,
+  posion,
+  rope,
+  trophy,
+];
```

Comment:
`weaponry` is a great word.  For arrays, I would suggest using plural words to name them.  That way, you can see implicitly know if the variable is an Array

`weapons` for instance helps us understand that there should be multiple, i.e. Array